conf/imperial.config

```diff
@@ -3,7 +3,7 @@
 params {
   // Config Params
   config_profile_description = 'Imperial College London - HPC Profile -- provided by nf-core/configs.'
-  config_profile_contact = 'Combiz Khozoie (c.khozoie@imperial.ac.uk)'
+  config_profile_contact = 'NA'
```

Comment:
Please just remove / comment out these lines if you don't want to be listed here any more.